packages/payload/src/admin/components/elements/ListDrawer/DrawerContent.tsx

```diff
@@ -146,7 +146,7 @@ export const ListDrawerContent: React.FC<ListDrawerProps> = ({
     let copyOfWhere = { ...(where || {}) }
 
     if (filterOptions) {
-      copyOfWhere = hoistQueryParamsToAnd(copyOfWhere, filterOptions[slug])
+      copyOfWhere = hoistQueryParamsToAnd(copyOfWhere, filterOptions[slug] || {})
```

Comment:
I'm not 100% sure if this would be an issue, but just re-check this to make sure:

`filterOptions[slug] || {}` seems to be useful for making sure `{}` is used instead of `null`.

But what if `filterOptions` is `false`? Shouldn't this stay as `false`? Maybe it should be `filterOptions[slug] !== null ? filterOptions[slug] : {}` instead

Reply:
I changed this a bit. The `null` scenario shouldn't be a factor here. I was more trying to dodge a typescript issue and pushed a more clear change for that.